src/const.js

```diff
@@ -162,6 +174,9 @@ const DefaultLabels = {
     allLabels.insect,
     allLabels.penguin,
     allLabels.sealion,
+    allLabels.deer,
+    allLabels.sheep,
+    allLabels.goat,
```

Comment:
Is ordering important here? If so, I'd probably put these 3 above penguin and sealion (and deer at the bottom of the 3).